docs/content/latest/yugabyte-platform/manage-deployments/remove-nodes.md

```diff
@@ -14,20 +14,74 @@ isTocNested: true
 showAsideToc: true
 ---
 
-## Remove an unresponsive node
-
-If a VM or machine in a universe reaches its end of life and has unrecoverable hardware or other system (OS, disk, etc.) issues, it will be detected and displayed in the Yugabyte Platform console as an `Unreachable` node. The Raft consensus algorithm will ensure that other leaders will get elected for the underlying data shards. But, the universe is in a partially under-replicated scenario and will not be able to tolerate many more failures. So a quick remedy is needed.
+If a virtual machine or a physical server in a universe reaches its end of life and has unrecoverable hardware or other system issues, such as problems with its operating system, disk, and so on, it is detected and displayed in the Yugabyte Platform UI as an unreachable node, as per the following illustration: 
 
 ![Unreachable Node Actions](/images/ee/node-actions-unreachable.png)
 
-So, for the `yb-15-node-actions-n3` node, you can use the **Actions** drop-down list at the end and choose the **Remove Node** option. This will bring the node to the `Removed` status and no further data can be accessed from this node as server processes are shut down.
+When this happens, new Master leaders are elected for the underlying data shards, but since the universe enters a partially under-replicated state, it would not be able to tolerate additional failures. To remedy the situation, the unreachable node needs to be eliminated by taking actions in the following sequence:
+
+- Step 1: [Remove node](#remove-node)
+- Step 2: [Release instance](#release-instance)
+- Step 3: [Delete node](delete-node)
+
+## Remove node
+
+The action to remove a node is available from the following node states:
```

Comment:
Is state the same as status? Should 'Unreachable' be in this list?

Reply:
Good catch, I'm going to check this

Reply:
I've got the answer of this and modified the doc